labs/fabric-operations-console.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Fabric Operations Console
+parent: labs
+---
+# Lab Name
+Fabric Operations Console
+
+# Short Description
+Hyperledger Fabric Operations Console code used in IBM's IBM Blockchain Platform Offering.
```

Comment:
I don't think the repetition is necessary.
```suggestion
Hyperledger Fabric Operations Console code used in the IBM Blockchain Platform Offering.
```

Reply:
done.  Thanks!